### PR TITLE
[DO NOT MERGE] Switch to ec2 and enable aarch64

### PR DESCRIPTION
**Pull request by lsm5**
[NO NEW TESTS NEEDED]

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

### CODE REVIEW

File: contrib/cirrus/setup.sh [hunk 1]
```diff
@@ -25,5 +25,5 @@ set -x  # show what's happening
 curl --fail --location -o /tmp/netavark.zip "$NETAVARK_URL"
 mkdir -p /usr/libexec/podman
 cd /usr/libexec/podman
-unzip /tmp/netavark.zip
-chmod a+x /usr/libexec/podman/netavark
+#unzip -q /tmp/netavark.zip
+#chmod a+x /usr/libexec/podman/netavark
```
Comment (cevich) [inline contrib/cirrus/setup.sh:29]: Yeah, this is definitely going to break on arm64, there is no task on netavark to build non-x86.  IIRC, there's an arm cross-compile task over there, perhaps we could just arrange for it to publish a build artifact for aardvark CI to use?

Comment (lsm5) [reply contrib/cirrus/setup.sh:29]: remind me, did we decide to get rid of the cross-compile task in favour of a separate aarch64 job like this one on netavark?

File: .cirrus.yml [hunk 2]
```diff
@@ -24,88 +24,99 @@ env:
...
     # This simply prevents rebuilding bin/aardvark-dns* or every subsequent task.
     folder: "$CIRRUS_WORKING_DIR/bin"
     fingerprint_key: "bin_v1_${CIRRUS_BUILD_ID}" # Cache only within same build
     reupload_on_changes: true
-  setup_script: &setup "$SCRIPT_BASE/setup.sh $CIRRUS_TASK_NAME"
-  main_script: &main "$SCRIPT_BASE/runner.sh $CIRRUS_TASK_NAME"
+
+setup_script: &setup "$SCRIPT_BASE/setup.sh $CIRRUS_TASK_NAME"
+
+main_script: &main "$SCRIPT_BASE/runner.sh $CIRRUS_TASK_NAME"
```
[the remark this answers is not in the document]
Comment (lsm5) [reply .cirrus.yml:45]: Now, if I have both `build` and `build_aarch64` I don't think both can do `main_script: &main "$SCRIPT_BASE/runner.sh $CIRRUS_TASK_NAME"`. There can be only one `&main` correct?

[the remark this answers is not in the document]
Comment (cevich) [reply .cirrus.yml:45]: Correct, and annoyingly neither git nor cirrus will complain loudly you if you do define an anchor more than once :cry:  The most fool-proof way I've found is to use the git `pre-commit` utility.  A quick and dirty way to check is to load it in python's `yaml` module (`yaml.safe_load(<file object>)`).

**Comment by lsm5** [comment]
just a trial run. closing in favour of https://github.com/containers/aardvark-dns/pull/141

**Pull request closed without merging**